tutorials/s4sdk-resilience/s4sdk-resilience.md

```diff
@@ -148,162 +181,165 @@ import javax.servlet.http.HttpServletResponse;
 
-import com.sap.cloud.sdk.cloudplatform.logging.CloudLoggerFactory;
-
 import com.sap.cloud.sdk.s4hana.datamodel.odata.namespaces.businesspartner.BusinessPartner;
 
 @WebServlet("/businesspartners")
 public class BusinessPartnerServlet extends HttpServlet {
 
     private static final long serialVersionUID = 1L;
-    private static final Logger logger = CloudLoggerFactory.getLogger(BusinessPartnerServlet.class);
+    private static final Logger logger = LoggerFactory.getLogger(BusinessPartnerServlet.class);
+
+    private static final String DESTINATION_NAME = "MyErpSystem";
 
     @Override
     protected void doGet(final HttpServletRequest request, final HttpServletResponse response)
             throws ServletException, IOException {
         try {
             final List<BusinessPartner> businessPartners =
-                    new GetBusinessPartnersCommand().execute();
-
+                    new GetBusinessPartnersCommand(DESTINATION_NAME).execute();
             response.setContentType("application/json");
             response.getWriter().write(new Gson().toJson(businessPartners));
-
         } catch (final Exception e) {
             logger.error(e.getMessage(), e);
             response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             response.getWriter().write(e.getMessage());
         }
     }
 }
+
 ```
+
 Thanks to our new `GetBusinessPartnersCommand`, we can now simply create a new command and execute it. As before, we get a list of business partners as result. But now we can be sure that our application will not stop working all-together if the OData service is temporarily unavailable for any tenant.
 
 
 [DONE]
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 4: ](Write tests for the Hystrix command)]
-There are two things we need to address in order to properly test our code: we need to provide our tests with an ERP endpoint and a `Hystrix` request context.
-
+[ACCORDION-BEGIN [Step 4: ](Write tests for the resilient command)]
 
-[DONE]
-[ACCORDION-END]
+There is one thing we need to address in order to properly test our code: we need to provide our tests with an ERP endpoint.
 
-[ACCORDION-BEGIN [Step 5: ](Credentials for SAP S/4HANA system)]
 We provide the ERP destination as explained in the [previous blog post] (https://blogs.sap.com/2017/05/21/step-4-with-sap-s4hana-cloud-sdk-calling-an-odata-service/) as a `systems.json` or `systems.yml` file. In addition, you may provide a `credentials.yml` file to reuse your SAP S/4HANA login configuration as described in the Appendix of the previous tutorial step.
 
-To provide credentials in this manner, create the following `credentials.yml` file in a safe location (e.g., like storing your ssh keys in ~/.ssh), i.e., not in the source code repository.
+To provide credentials in this manner, create the following `credentials.yml` file (if you have not already) in a safe location (e.g., like storing your ssh keys in ~/.ssh), i.e., not in the source code repository.
```

Comment:
Shouldn't we only and directly refer to the previous tutorial instead of repeating this here?

Reply:
Updated